docs/tags.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: sub-navigation
+order: 7
+title: Adding tags
+description: Use tags to categorise posts and make it easier for readers to browse content on your site.
+---
+
+The GOV.UK Eleventy Plugin makes it easy to add tags to your website. Follow these instructions to enable this feature.
+
+## Create a tags page
+
+To include a list of all tags used on your website, create a page that uses the `tags` layout:
+
+```yaml
+---
+eleventyExcludeFromCollections: true
+layout: tags
+---
+```
+
+You can see an [example of a tag index](../layouts/tags) on this website.
+
+## Create a tag listing page
+
+To generate a page for each tag used on your website, create a page that uses the `tag` layout and paginates `collections.tags` data:
+
+```yaml
+---
+eleventyExcludeFromCollections: true
+layout: tag
+pagination:
+  data: collections.tags
+  size: 1
+  alias: tag
+permalink: "/tags/{% raw %}{{ tag | slug }}{% endraw %}/"
```

Comment:
Could we do something like this to set the title here?

```suggestion
permalink: "/tags/{% raw %}{{ tag | slug }}{% endraw %}/"
title: Posts tagged ‘{{ tag }}’
```

Reply:
You would need to use `eleventyComputed.title`, but that blew up (see above comment).

Reply:
Title for tag pages is now configurable. If you are using breadcrumbs, and want to show them on tag pages, you need to add `eleventyComputed.title`. If not, you can still add a title, otherwise it will default to `Posts tagged ‘[tag]’`.